dluhc-component-library/src/components/siteSelectionForm/db.ts

```diff
@@ -0,0 +1,22 @@
+import Dexie from "dexie";
+import { FormState } from "./types";
+
+interface IFormData {
+  id?: string;

```

Comment:
Why is `id` optional? I would have expected that to be required.

Reply:
Optional `id` was leftover from a previous commit which used an auto-increment for the id. Have now removed.